packages/scxa-faceted-search-results-v5/src/FilterList.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import _ from 'lodash'
+class FilterList extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      ascending: false,
+      sortTitle: props.sortTitle
+    }
```

Comment:
I think that for the time being we should remove sorting. Having pagination makes it really unintuitive. As a side benefit, we can have a stateless `FilterList`.